exist-jetty-config/src/main/resources/standalone-webapp/WEB-INF/web.xml

```diff
@@ -9,9 +9,9 @@
 <web-app 
     xmlns="http://xmlns.jcp.org/xml/ns/javaee" 
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee https://www.oracle.com/webfolder/technetwork/jsc/xml/ns/javaee/web-app_4_0.xsd"
     metadata-complete="false"
-    version="3.1">
+    version="5.0">
```

Comment:
I think this should refer to the Jakarta schemas from now on. See https://jakarta.ee/xml/ns/jakartaee/ for more details on this.

Reply:
@reinhapa Good shout! I have pushed an additional commit that addresses this.